suite-native/device-manager/src/components/WalletItem.tsx

```diff
@@ -54,6 +58,9 @@ const labelStyle = prepareNativeStyle(() => ({
 export const WalletItem = ({ deviceState, onPress, isSelectable = true }: WalletItemProps) => {
     const { applyStyle } = useNativeStyles();
     const device = useSelector((state: any) => selectDeviceByState(state, deviceState));
+    const hasAccounts = useSelector((state: any) =>
+        device?.state ? selectDeviceHasAccountsByDeviceState(state, device.state) : false,
```

Comment:
Could you please put the check for device.state existence inside the selector itself? 

And I don't like the `any` type usage in this file. I know it doesn't add a much type safety to use root states in useSelector, but it's easy to do (you can import all the root state types besides the selectors) and it's always good to avoid `any` if possible.